layers/+lang/php/funcs.el

```diff
@@ -13,3 +13,14 @@
   "Conditionally setup php backend."
   (pcase php-backend
     (`lsp (lsp))))
+
+(defun spacemacs//php-setup-dap ()
+  "Conditionally setup elixir DAP integration."
+  ;; currently DAP is only available using LSP
+  (pcase (spacemacs//php-backend)
```

Comment:
Is this supposed to be:
```suggestion
  (pcase php-backend
```
Then it would match the way the backend is checked in the `spacemacs//php-setup-backend` function on line 14 above?

Because I'm not finding any definition of a function that's called: `spacemacs//php-backend`

Reply:
Fixed in 491d0a4fdd256c466717e723e69b735ca4c92a74 buuuuut...
Actually it was working fine before a fix.

Silly but if You ask me what exactly was working - unfortunately I don't have an answer. I've copypasted the code from python layer and it seems like python and php layers don't follow same conventions so 'spacemacs//' prefix is odd for php..

So actually I don't know whether this code is needed at all if dap-php works fine both before and after fix. Anyway for python it probably does smth if exists. Similar code exists in other layers too AFAIR.

Reply:
The function name conventions are:
https://github.com/syl20bnr/spacemacs/blob/master/doc/CONVENTIONS.org#spacemacs-core-and-layer
- spacemacs/xxx is an interactive function called xxx
- spacemacs//xxx is a private function called xxx (implementation details)
- spacemacs|xxx is a macro called xxx

The changelog is in the right place and it looks fine, it's just missing the thanks parentheses:
`(thanks to <your name>)`, I know it's strange to thank yourself 😄 but that's the convention, and it helps when the PR author already has added it.

The `spacemacs//php-setup-dap` function isn't used anywhere, so currently it has no effect.

Could you squash the commits, it's one commit per PR (unless it's a complex PR).

Reply:
Thank's for a link to conventions, it will be useful.
I'll re-check and fix everything and squash commits.